File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/DataGridView/PopulatingData/HierarchiesPage.vb
Namespace CompanionKit.Controls.DataGridView.PopulatingData

    Public Class HierarchiesPage

        Public Sub New()

            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()



        End Sub

        ''' <summary>
        ''' Handles the Load event of the HierarchiesPage control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub HierarchiesPage_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
            'Fill DataGridView
            Me.ordersAdapter.Fill(mobjDS.Orders)
            Me.orderDetailsAdapter.Fill(Me.mobjDS.Order_Details)
            Me.productsAdapter.Fill(Me.mobjDS.Products)
        End Sub
    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/DataGridView/PopulatingData/HierarchiesPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.DataGridView.PopulatingData
{
    public partial class HierarchiesPage : UserControl
    {
        public HierarchiesPage()
        {
            InitializeComponent();
            
        }

        /// <summary>
        /// Handles the Load event of the HierarchiesPage control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void HierarchiesPage_Load(object sender, EventArgs e)
        {
            //Fill DataGridView

            this.ordersAdapter.Fill(mobjDS.Orders);
            this.orderDetailsAdapter.Fill(this.mobjDS.Order_Details);
            this.productsAdapter.Fill(this.mobjDS.Products);
        }
    }
}